ui/app/models/task.js

```diff
@@ -50,4 +50,27 @@ export default class Task extends Fragment {
   @attr('number') reservedEphemeralDisk;
 
   @fragmentArray('volume-mount', { defaultValue: () => [] }) volumeMounts;
+
+  async _fetchParentJob() {
+    let job = await this.store.findRecord('job', this.taskGroup.job.id, {
+      reload: true,
+    });
+    this._job = job;
```

Comment:
`question`: I'm a little confused by why we're setting a private variable on the `Task` fragment. The `Task` fragment is owned by the `TaskGroup` fragment and that is stored (and owned) on our `Job` model. As a result of this relationship, shouldn't we already have access to the owner (the job)?